### ISSUE TITLE
Build docker image of the studio components

### ISSUE THREAD

**Issue by amalkurup89** [Apicurio/apicurio-studio#1453]
Is there any reference document, which I can refer to build docker image of the below studio components

a) studio-ui
b) studio-api
c) studio-ws

**Comment by EricWittmann** [comment]
I don't think we specifically have documentation for how to build the docker images, but you can see how it's done in the release workflow:

https://github.com/Apicurio/apicurio-studio/blob/master/.github/workflows/release.yaml#L109-L114

Obviously you would need to do that after a local build.

**Comment by carlesarnal** [comment]
Maybe these three readmes are good candidates or a good place to start.

- https://github.com/Apicurio/apicurio-studio/blob/master/platforms/thorntail/api/README.md
- https://github.com/Apicurio/apicurio-studio/blob/master/platforms/thorntail/ui/README.md
- https://github.com/Apicurio/apicurio-studio/blob/master/platforms/thorntail/ws/README.md

As Eric says you would need to do that after a local build.
